node/libs/roles/src/proto/validator.proto

```diff
@@ -12,6 +12,7 @@ message Fork {
 message Genesis {
   optional Fork fork = 1; // required
   repeated PublicKey validators = 2;
+  repeated BatchPublicKey attesters = 3;
```

Comment:
@pompon0 @brunoffranca This is what I'm trying to do in the `Genesis` message to add keys from the attesters but it's failing

Reply:
you need to add `import zksync/roles/attester.proto` and use relative path `attester.BatchPublicKey`.